DQMOffline/Trigger/plugins/JetMonitor.cc

```diff
@@ -0,0 +1,594 @@
+   std::string isDeno = "";
+   isDeno = denu;
+   std::string DenoOrNume = "";
+   DenoOrNume = denu;
+
+   if (DenoOrNume == "denominator")
+   {
+      // index 0 = pt, 1 = vari.pt , 2 = pt vs ls , 3 = phi, 4 = eta, 5 = eta vs phi , 6 = eta vs pt
+      a_me[0].denominator->Fill(pt_);// pt
+      a_me[1].denominator->Fill(pt_);// jetpT Threshold binning for pt 
+      a_me[2].denominator->Fill(ls_,pt_);// pt vs ls
+      a_me[3].denominator->Fill(phi_);// phi 
+      a_me[4].denominator->Fill(eta_);// eta
+      a_me[5].denominator->Fill(eta_,phi_);// eta vs phi
+      a_me[6].denominator->Fill(eta_,pt_);// eta vs pT
+   }
+   else if (DenoOrNume == "numerator")
+   {
+      a_me[0].numerator->Fill(pt_);// pt 
+      a_me[1].numerator->Fill(pt_);// jetpT Threshold binning for pt 
+      a_me[2].numerator->Fill(ls_,pt_);// pt vs ls
+      a_me[3].numerator->Fill(phi_);// phi
+      a_me[4].numerator->Fill(eta_);// eat
+      a_me[5].numerator->Fill(eta_,phi_);// eta vs phi
+      a_me[6].numerator->Fill(eta_,pt_);// eta vs pT 
+   }
+   else {
+   std::cout << "CHECK OUT denu option in FillME !!!"<< std::endl;
+   }
+}
+void JetMonitor::bookMESub(DQMStore::IBooker & Ibooker , JetME* a_me,const int len_,std::string h_Name ,std::string h_Title, std::string h_subOptName , std::string h_suOptTitle ){
+   std::string hName = h_Name;
+   std::string hTitle = h_Title;
+   std::string hSubN =""; 
+   std::string hSubT =""; 
+   hSubT = h_suOptTitle;
+
+   int nbin_phi = jet_phi_binning_.nbins;
+   double maxbin_phi = jet_phi_binning_.xmax;
+   double minbin_phi = jet_phi_binning_.xmin;
+
+   int nbin_eta = jet_eta_binning_.nbins;
+   double maxbin_eta = jet_eta_binning_.xmax;
+   double minbin_eta = jet_eta_binning_.xmin;
+
+   if (h_subOptName != ""){
+      hSubN = "_"+h_subOptName;
+   }
+
+   if (h_subOptName == "HEP17") {
+      nbin_phi = phi_binning_hep17_.nbins;
+      maxbin_phi = phi_binning_hep17_.xmax;
+      minbin_phi = phi_binning_hep17_.xmin;
+
+      nbin_eta = eta_binning_hep17_.nbins;
+      maxbin_eta = eta_binning_hep17_.xmax;
+      minbin_eta = eta_binning_hep17_.xmin;
+   }
+   if (h_subOptName == "HEM17") {
+      nbin_phi = phi_binning_hep17_.nbins;
+      maxbin_phi = phi_binning_hep17_.xmax;
+      minbin_phi = phi_binning_hep17_.xmin;
+
+      nbin_eta = eta_binning_hem17_.nbins;
+      maxbin_eta = eta_binning_hem17_.xmax;
+      minbin_eta = eta_binning_hem17_.xmin;
+   }
+   if (h_subOptName == "HEP18") {
+      nbin_phi = phi_binning_hep18_.nbins;
+      maxbin_phi = phi_binning_hep18_.xmax;
+      minbin_phi = phi_binning_hep18_.xmin;
+
+      nbin_eta = eta_binning_hep17_.nbins;
+      maxbin_eta = eta_binning_hep17_.xmax;
+      minbin_eta = eta_binning_hep17_.xmin;
+   }
+   hName = h_Name+"pT"+hSubN;
+   hTitle = h_Title+" pT " + hSubT;
+   bookME(Ibooker,a_me[0],hName,hTitle,met_binning_.nbins,met_binning_.xmin, met_binning_.xmax);
```

Comment:
I didn't changed the binning variable name. 
At here, I used met_binning, It's for Jet pT. 
If you want that I change this name, I will change it.